pandas/core/generic.py

```diff
@@ -3156,6 +3156,13 @@ def copy(self, deep=True):
         data = self._data.copy(deep=deep)
         return self._constructor(data).__finalize__(self)
 
+    __copy__ = copy
```

Comment:
@bmcfee but if a class (`NDFrameChild`) inherits from `NDFrame` and overrides `.copy`: with the proposed code `NDFrameChild.__copy__` remains `NDFrame.copy`, rather than `NDFFrameChild.copy`